framework/src/base/FEProblem.C

```diff
@@ -3582,11 +3573,10 @@ FEProblem::adaptMesh()
   unsigned int cycles_per_step = _adaptivity.getCyclesPerStep();
   for (unsigned int i=0; i < cycles_per_step; ++i)
   {
-    _console << "Adaptivity step " << i+1 << " of " << cycles_per_step << "\n";
+    _console << "Adaptivity step " << i+1 << " of " << cycles_per_step << std::endl;
     if (_adaptivity.adaptMesh())
       meshChanged();
   }
-  _console << std::flush;
```

Comment:
Hmm so you added a std::endl above in a loop and removed the flush here outside of the loop.